code/modules/mob/living/living.dm

```diff
@@ -244,14 +244,18 @@ Sorry Giacom. Please don't be mad :(
 
 /mob/living/proc/adjustBruteLoss(var/amount)
 	if(status_flags & GODMODE)	return 0
-	bruteloss = min(max(bruteloss + amount, 0),(maxHealth*2))
+	bruteloss += amount

```

Comment:
You've removed the Clamp() (what was there before is equivalent to Clamp()), when you really only needed to remove the cap.

change it to bruteloss = max(0, bruteloss+amount), instead of checking if it's below 0 seperately